js/components/ReportSummaryForm.vue

```diff
@@ -16,17 +16,17 @@
       </ul>
       <div class="row form-group">
         <div class="col">
-          <label for="title">Report Count Title</label>
-          <input id="title" name="title" v-model.trim="model.title" type="text" class="form-control" :class="{ 'is-invalid': errors.title }">
+          <label :for="'title' + model.id">Report Count Title</label>
+          <input :id="'title' + model.id" name="title" v-model.trim="model.title" type="text" class="form-control" :class="{ 'is-invalid': errors.title }">
```

Comment:
Seems like you could extract a helper method here and use it to construct the IDs:

```
methods: {
  fieldId(name) {
    const { model } = this;
    return `name${model.id}`;
  }
  // ...
}
```

```
<label :for="fieldId('title')">Report Count Title</label>
<input :id="fieldId('title')" name="title" v-model.trim="model.title" type="text" class="form-control" :class="{ 'is-invalid': errors.title }">
```

Reply:
Or you could just concatenate the index into the name (`fieldId('name' + index)`). Either way works for me.